packages/generator-volto/generators/app/templates/README.md

```diff
@@ -1,6 +1,6 @@
 ## Documentation
 
-A training on how to create your own website using Volto is available as part of the Plone training at [https://training.plone.org/5/volto/index.html](https://training.plone.org/5/volto/index.html).
+A training on how to create your own website using Volto is available as part of the Plone training at [https://2022.training.plone.org/volto/index.html](https://2022.training.plone.org/volto/index.html).
```

Comment:
I apologize for the last-minute change of plans. We decided that the default Volto training should be the following.

```suggestion
[Volto Hands-On](https://training.plone.org/voltohandson/index.html) is a training on how to create your own website.
```